### PR TITLE
@thunderstore/cyberstorm: refactor Connections component

### PR DESCRIPTION
**Pull request by anttimaki**

### CODE REVIEW

**Comment by anttimaki** [comment]
Current dependencies on/for this PR:
* master
  * **PR #832** <a href="https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/832" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/thunderstore-io/thunderstore-ui/832?utm_source=stack-comment).

**Review by MythicManiac — APPROVED** [review]
LGTM

File: packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.tsx [hunk 1]
```diff
@@ -1,70 +1,29 @@
...
+import { SettingItem } from "../../../SettingItem/SettingItem";
+import { Switch } from "../../../Switch/Switch";
 
-export interface ConnectionsProps {
-  userData: UserSettings;
-}
+const PROVIDERS = [
+  { name: "Discord", icon: faDiscord },
+  { name: "GitHub", icon: faGithub },
+  { name: "Overwolf", icon: faGithub }, // TODO: icon
```
Comment (Oksamies) [inline packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.tsx:15]: We have `OverwolfLogo` in `packages\cyberstorm\src\svg\svg.tsx`

**Review by Oksamies — CHANGES_REQUESTED** [review]

File: packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.tsx (hunk 1 above)
Comment (anttimaki) [reply packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.tsx:15]: Yeah and no icon wrapper so I'll consider that outside the scope of the PR.

Comment (Oksamies) [reply packages/cyberstorm/src/components/Layout/Settings/Connections/Connections.tsx:15]: I have a open PR for adding wrapper for all the Icons, so mentally I'm not going accept this as a answer. But I'm still going to accept the review and add it myself 👍

**Review by Oksamies — APPROVED** [review]

**Pull request merged**